sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -10,6 +10,117 @@ import { KeyCredential } from '@azure/core-auth';
 import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
 
+// @public
+export interface ACSChatEventBaseProperties {
```

Comment:
Do you need the "BaseProperties" suffix here?

Reply:
I removed the needless "Properties" suffix from these types.